packages/cli/oclif.manifest.json

```diff
@@ -862,6 +862,15 @@
           "hidden": false,
           "name": "verbose",
           "type": "boolean"
+        },
+        "watch": {
+          "allowNo": false,
+          "char": "w",
+          "description": "Re-run the function when the source code changes.",
+          "env": "SHOPIFY_FLAG_WATCH",
+          "hidden": false,
+          "name": "watch",
+          "type": "boolean"
```

Comment:
Question: does this get auto generated or we manually add it based on the app/command/replay file?